### PR TITLE
save score working

### PR DESCRIPTION
**Pull request by yuna-iwata**
-added a submit score function in networking
- can now submit score to the backend

### CODE REVIEW

File: src/Pages/GamePage.jsx [hunk 1]
```diff
@@ -31,18 +32,20 @@ export default function Gamepage({ game, setGame }) {
...
   }, []);
 
-  const handleClick = () => {
+  const handleSendClick = async () => {
     let scene = game.scene.keys.helloworld;
     const handlescore = scene.createScore();
     console.log(handlescore);
     setScore(handlescore);
+    const response = await submitScore(handlescore, currentUser);
+    console.log(response);
```
Comment (louisglan) [inline src/Pages/GamePage.jsx:41]: this needs removing

File: src/Networking.js [hunk 2]
```diff
@@ -80,3 +80,22 @@ export async function changeUsername(oldUsername, newUsername, password) {
     console.log(e.message);
   }
 }
+
+export async function submitScore(score, username) {
+  try {
+    console.log(score, username);
```
Comment (louisglan) [inline src/Networking.js]: This console.log needs removing

File: src/Pages/GamePage.jsx (hunk 1 above)
Comment (louisglan) [reply src/Pages/GamePage.jsx:41]: as in the console.log

Comment (yuna-iwata) [reply src/Pages/GamePage.jsx:41]: cool done

**Review by louisglan — APPROVED** [review]
Looks great

**Pull request merged**